module_4/task_1/free_disk_space.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+threshold=1000
+
+checkFreeSpace() {
+  while true
+  do
+      local freeSpace=$(free --mega | grep 'Mem' | awk '{print $4}')
```

Comment:
The `free` command is used to get the amount of free memory, which is not directly related to the amount of free disk space, try using `df`. command instead.